apps/back-end/.editorconfig

```diff
@@ -0,0 +1,20 @@
+# Cross-platform formatting config
```

Comment:
We can remove this file since it exists in the root directory

Reply:
This was added when it was a stand-alone project. So I'm ok removing this, for now at least.

But more generally, I was wondering what happens any of these subprojects need to become git-subtree inclusions later, so have their own self-standing repos.  It seems like those repos may have lost important definitions in the interest of de-duplication. Something to be wary of, a point Victor DevOps-Toolkit makes here about [monorepos encouraging [good/bad] tight coupling](https://youtu.be/VvcJGjjEyKo?t=499).

Reply:
Yeah I think it makes sense not tightly couple the repos unnecessarily, especially in the actual code. I'm just thinking for files like this, which are mainly there for our developability, there's a risk of having duplicated files in different places that could get out of sync and lead to different codestyles.